daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md

```diff
@@ -129,6 +129,47 @@ This component supports **output binding** with the following operations:
 
 - `create`: publishes a message to the specified queue
 
+## Message metadata
+
+Azure Service Bus messages extend the Dapr message format with additional contextual metadata. Some metadata fields are set by Azure Service Bus itself (read-only) and others can be set by the client when publishing a message through `Invoke` binding call with `create` operation.
+
+### Sending a message with metadata
+
+To set Azure Service Bus metadata when sending a message, set the query parameters on the HTTP request or the gRPC metadata as documented [here]({{< ref "pubsub_api.md#metadata" >}}).
```

Comment:
@ItalyPaleAle just a quick clarification, I believe we can set the metadata in the bindings request similar to how we set for pubsub right?

Reply:
Yes it should be similar: https://docs.dapr.io/reference/api/bindings_api/#invoking-output-bindings

Reply:
Although that link there is for pubsub and should eb changed to the bindings one